src/DataTable/Table.js

```diff
@@ -65,7 +65,7 @@ class Table extends React.Component {
                 </thead>
                 <tbody>
                     {realRows.map((row, idx) =>
-                        <tr key={row[rowKeyColumn] || row.key || idx} className={row.className}>
+                        <tr key={row[rowKeyColumn] || row.key || idx} className={row.className} style={row.style}>
```

Comment:
So do I... ;)

However, in the meantime, since there is support for className, I believe it only makes sense to have support for style, even if it means handling the entire thing differently later on.

What checks on the nature of 'style' should be performed to make it safe?
What checks on 'className' ?
Also, maybe 'style' should be renamed to something less likely to collide, like 'rowStyle' ?